Themes/default/BoardIndex.template.php

```diff
@@ -178,6 +132,140 @@ function template_main()
 }
 
 /**
+ * Outs the board icon for a standard board.
```

Comment:
Nit: "Outs the..." is weird English. This should probably be "Outputs the..." or (better) "Prints the..."